web/src/components/netflow-traffic-parent.tsx

```diff
@@ -23,10 +28,25 @@ class NetflowTrafficParent extends React.Component<Props, State> {
     return { error: err };
   }
 
+  componentDidMount() {
+    getRole()
+      .then(role => {
+        let namespace = this.state.namespace;
+        if (role === 'dev') {
+          namespace = window?.sessionStorage?.getItem('bridge/last-namespace-name') || 'default';
```

Comment:
what's this 'bridge/last-namespace-name' ? Isn't it specific to running in dev mode with the bridge?